### ISSUE TITLE
list index out of range

### ISSUE THREAD

**Issue by gnebbia** [PaulSec/API-dnsdumpster.com#24]
If I try to execute these two instructions:
```python
from dnsdumpster.DNSDumpsterAPI import DNSDumpsterAPI
results = DNSDumpsterAPI().search("example.com")
````
I get the following error "list index out of range", I am using the latest version of the dnsdumpster module.

**Comment by tensts** [comment]
I just fix that in pull request #25

**Comment by gnebbia** [comment]
Ok thanks!
